org.jacoco.core.test/src/org/jacoco/core/test/validation/ExceptionsTest.java

```diff
@@ -103,6 +103,13 @@ public void testCoverageResult() {
 		assertLine("explicitExceptionFinally.finallyBlock",
 				ICounter.FULLY_COVERED);
 
+		// TODO(Godin): why fully covered? it looks strange, but the same in
```

Comment:
@Godin Fully covered only in respect to instructions. But 1 branch missed, 1 branch covered. Makes sense?